drivers/display/display_ili9xxx.c

```diff
@@ -163,6 +163,7 @@ static int ili9xxx_write(const struct device *dev, const uint16_t x,
 	mipi_desc.width = desc->width;
 	/* Per MIPI API, pitch must always match width */
 	mipi_desc.pitch = desc->width;
+	mipi_desc.frame_incomplete = desc->frame_incomplete;
```

Comment:
Just FYI- we can also add this to the ST7796S display driver

Reply:
I'm not sure what exactly to change here, I just tried to fix it to my best knowledge. Feel free to tell me exactly where to change things

Reply:
This is the right place :) I was just mentioning that other drivers like the ST7796 can use this as well. It is definitely a nonblocking suggestion, this can always be added after the fact. One thing we probably should do though is zero-initialize the `mipi_desc` structures in these drivers in places they aren't already.

Reply:
Yah, that's why this change is there. I didn't initially find more, so if you have more, feel free to tell me.
The others I found are already zero initialized as previously discussed.

Reply:
@danieldegrasse Fixed